lib/controller/maalcie/BeheerAbonnementenController.class.php

```diff
@@ -12,8 +12,13 @@
  */
 class BeheerAbonnementenController extends AclController {
 
+    /**
+     * @var MaaltijdAbonnementenModel
+     */
+    protected $model;
+
 	public function __construct($query) {
-		parent::__construct($query, null);
+		parent::__construct($query, MaaltijdAbonnementenModel::transaction());
```

Comment:
FYI: individuele method calls naar het model zijn transactions, dus 2x $this->model->iets() na elkaar roepen zijn 2 aparte transacties.